src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarBidAction.test.tsx

```diff
@@ -364,51 +378,89 @@ describe("ArtworkSidebarBidAction", () => {
         expect(wrapper.text()).toContain("$900")
         expect(wrapper.text()).toContain("Bid")
       })
+
+      it("does not display anything about identity verification even if sale requires it and user is not identity verified", async () => {
+        const artwork = merge(
+          ArtworkFromTimedAuctionRegistrationClosed,
+          NotRegisteredToBid,
+          { sale: SaleRequiringIDV }
+        )
+
+        const wrapper = await getWrapper({
+          artwork,
+          me: NotIDVedUser,
+        })
+
+        expect(wrapper.text()).not.toContain(
+          "Identity verification required to bid."
+        )
+      })
     })
   })
 
   describe("for live auction", () => {
-    it("with open registration and not registered bidder ", async () => {
-      const artwork = merge(
-        ArtworkFromLiveAuctionRegistrationOpen,
-        NotRegisteredToBid
-      )
-      const wrapper = await getWrapper({ artwork, me: NotIDVedUser })
+    describe("when user is not registered to bid and registration is open", () => {
+      it("displays that the user can enter live bidding", async () => {
+        const artwork = merge(
+          ArtworkFromLiveAuctionRegistrationOpen,
+          NotRegisteredToBid
+        )
+        const wrapper = await getWrapper({ artwork, me: NotIDVedUser })
 
-      expect(wrapper.text()).toContain("Enter live bidding")
+        expect(wrapper.text()).toContain("Enter live bidding")
+      })
     })
 
-    it("with closed registration and not registered bidder ", async () => {
-      const artwork = merge(
-        ArtworkFromLiveAuctionRegistrationClosed,
-        NotRegisteredToBid
-      )
+    describe("when user is not registered to bid and registration is closed", () => {
+      it("displays that registration is closed, but user can watch live bidding", async () => {
+        const artwork = merge(
+          ArtworkFromLiveAuctionRegistrationClosed,
+          NotRegisteredToBid
+        )
 
-      const wrapper = await getWrapper({ artwork, me: NotIDVedUser })
+        const wrapper = await getWrapper({ artwork, me: NotIDVedUser })
 
-      expect(wrapper.text()).toContain("Registration closed")
-      expect(wrapper.text()).toContain("Watch live bidding")
+        expect(wrapper.text()).toContain("Registration closed")
+        expect(wrapper.text()).toContain("Watch live bidding")
+      })
     })
 
-    it("with bidder registration pending approval", async () => {
-      const artwork = merge(
-        ArtworkFromLiveAuctionRegistrationOpen,
-        BidderPendingApproval
-      )
+    // TODO: This behavior that this test is testing seems wrong. If the bidder
+    // is pending approval, they won't be qualified for bidding and therefore can't
+    // bid yet...
+    describe("when the user is pending bidder approval and registration is still open", () => {
```

Comment:
Should this still include the line that it's applicable only when 'live auction' is in progress. I think as we saw yesterday before the live auction is open in this case they will see ' Registration pending'?

Reply:
Note the [`describe("for live auction", ...)`](https://github.com/artsy/reaction/pull/3345/files#diff-89ddabd613b2ace13b6023b270de6daeR404) above. Doe that clarify?